### PR TITLE
PX-1434: adding an extra check in the case of not having tests

### PR DESCRIPTION
**Pull request by GeylaniBerk**

### CODE REVIEW

**Review by paolo-rechia — CHANGES_REQUESTED** [review]
I'd change this to give us a more meaningful messages:

1.`Unit tests skipped because serverless file was not found.`
2.`Unit tests skipped because "skipUnitTests" flag was turned on.`

File: eb7_sls_helper/src/gh_action_interface.py [hunk 1]
```diff
@@ -347,9 +347,11 @@ def read_serverless_config(filename):
             set_output(f"formatted", "Integration tests are being skipped")
     elif inputs["mode"] == "tox":
         skip_unit_tests = eb7_helper_config.get("skipUnitTests", False)
-        if not skip_unit_tests:
+        if not skip_unit_tests and sls:
```
Comment (maxim-ebot7) [inline eb7_sls_helper/src/gh_action_interface.py:350]: Why do we check the sls file?(serverless.yml)

Should we check tox.ini is missing instead? Or what's the flow here?

Comment (paolo-rechia) [reply eb7_sls_helper/src/gh_action_interface.py:350]: That could also work. Missing serverless file could skip everything, right?

Comment (paolo-rechia) [reply eb7_sls_helper/src/gh_action_interface.py:350]: Now that you mention, we should also fix this for integration tests. We can move the check a few lines above, before trying to run any of these blocks, I think. Except maybe for linter?

Comment (GeylaniBerk) [reply eb7_sls_helper/src/gh_action_interface.py]: I added check before deploy, integration test and unit test

**Review by paolo-rechia — APPROVED** [review]
LGTM

File: eb7_sls_helper/src/gh_action_interface.py [hunk 2]
```diff
@@ -344,12 +346,12 @@ def read_serverless_config(filename):
         if not skip_integration_test:
             test(sls, inputs, args)
         else:
-            set_output(f"formatted", "Integration tests are being skipped")
+            set_output(f"formatted", "Integration tests skipped because 'skipIntegrationTests'' flag was turned on.")
```
Comment (zvonimir-ebot7) [inline eb7_sls_helper/src/gh_action_interface.py:349]: I would move these checks out of the `__name__ == "__main__"` block, which is already too complex.

**Pull request merged**